tests/pc/test_lag_member.py

```diff
@@ -0,0 +1,360 @@
+            if len(dut_ports[ATTR_PORT_BEHIND_LAG]) == number_of_lag_member:
+                dut_ports[ATTR_PORT_NOT_BEHIND_LAG] = {
+                    "port_id": port_id,
+                    "port_name": port
+                }
+                break
+            dut_ports[ATTR_PORT_BEHIND_LAG][port_id] = port
+
+    pytest_require(len(dut_ports[ATTR_PORT_BEHIND_LAG]) == number_of_lag_member and len(dut_ports[ATTR_PORT_NOT_BEHIND_LAG]), "No port for testing")
+
+    ptf_ports = {
+        ATTR_PORT_BEHIND_LAG: {},
+        ATTR_PORT_NOT_BEHIND_LAG: {}
+    }
+
+    for port_id in ptf_ports_available_in_topo:
+        if port_id in dut_ports[ATTR_PORT_BEHIND_LAG]:
+            ptf_ports[ATTR_PORT_BEHIND_LAG][port_id] = ptf_ports_available_in_topo[port_id]
+        elif port_id == dut_ports[ATTR_PORT_NOT_BEHIND_LAG]["port_id"]:
+            ptf_ports[ATTR_PORT_NOT_BEHIND_LAG] = {
+                "port_id": port_id,
+                "port_name": ptf_ports_available_in_topo[port_id]
+            }
+
+    vlan = {
+        "id": 109,
+        "ip": "192.168.9.1/24"
+    }
+    dut_lag_map = setup_dut_lag(duthost, dut_ports, vlan)
+    ptf_lag_map = setup_ptf_lag(ptfhost, ptf_ports, vlan)
+    return dut_lag_map, ptf_lag_map
+
+@pytest.fixture(scope="module")
+def common_setup_teardown(ptfhost):
+    logger.info("########### Setup for lag testing ###########")
+
+    ptfhost.shell("mkdir -p {}".format(TEST_DIR))
+    # Copy PTF test into PTF-docker for test LACP DU
+    test_files = ["lag_test.py", "acs_base_test.py", "router_utils.py"]
+    for test_file in test_files:
+        src = "../ansible/roles/test/files/acstests/%s" % test_file
+        dst = TEST_DIR + test_file
+        ptfhost.copy(src=src, dest=dst)
+
+    yield ptfhost
+
+    ptfhost.file(path=TEST_DIR, state="absent")
+
+@pytest.fixture(scope="module")
+def ptf_dut_setup_and_teardown(duthost, ptfhost, tbinfo):
+    """
+    Setup and teardown of ptf and dut
+
+    Args:
+        duthost: DUT host object
+        ptfhost: PTF host object
+        tbinfo: fixture provides information about testbed
+    """
+    dut_lag_map, ptf_lag_map = setup_dut_ptf(ptfhost, duthost, tbinfo)
+
+    yield dut_lag_map, ptf_lag_map
+
+    dut_teardown(duthost, dut_lag_map)
+    ptf_teardown(ptfhost, ptf_lag_map)
+
+def get_port_channel_status(duthost, port_channel):
+    """
+    Get status of port channel by port channel name
+
+    Args:
+        duthost: DUT host object
+        port_channel: port channel name
+
+    Returns:
+        status of port channel
+    """
+    commond_output = duthost.shell("docker exec -i teamd teamdctl {} state dump".format(port_channel))
+    pytest_assert(commond_output["rc"] == 0, "Get status of {}".format(port_channel))
+    json_info = json.loads(commond_output["stdout"])
+    return json_info
+
+def test_lag_member_status(duthost, ptf_dut_setup_and_teardown):
+    """
+    Test ports' status of 16 members in a lag
```

Comment:
This is not accurate.
Your verification is based on `HWSKU_INTF_NUMBERS_DICT` and you will use 8 if the HwSku is not registered

Reply:
Fixed